scripts/rhoso_adoc_docs_to_text.py

```diff
@@ -19,6 +19,7 @@
 from pathlib import Path
 import logging
 from lightspeed_rag_content.asciidoc import AsciidoctorConverter
+from packaging.version import Version
```

Comment:
It looks like we have a new dependency. The `requirements.txt` should be updated [1].

[1] https://github.com/openstack-lightspeed/rag-content/blob/main/requirements.txt

Reply:
I though this was part of the standard libraries, but I was wrong. I'll push a change through momentarily.